deploy.sh

```diff
@@ -84,6 +84,8 @@ build_push_images() {
 
 if [ "${TRAVIS_BRANCH}" == 'master' ]; then
```

Comment:
remove master from here as we dont need it

Reply:
sure.. done @Madhu-1